playwright/e2e/organizations/organizations-settings-team-members.spec.ts

```diff
@@ -335,9 +335,11 @@ test.describe('organizations settings team members page', () => {
         // It lets the owner user change the role of the other users.
         await memberListItem.getByRole('button', { name: /member/i }).click();
         const role = roleIterator.next().value!;
+
         await page
           .getByRole('option', { name: role })
           .getByRole('button')
+          .first()
```

Comment:
Wait, is this intentional? Are we expecting two buttons? If yes, why is only the first one relevant?